.changeset/loud-students-bake.md

```diff
@@ -0,0 +1,65 @@
+---
+"@thirdweb-dev/sdk": minor
+"@thirdweb-dev/react": minor
```

Comment:
split this into 2 changesets, one for SDK one for `react-core` (since the useContractRead and useContractWrite hooks are coming from react-core (and will also affect react-native) - changesets will do that automatically if u just change react-core.

each changeset should clearly state the changes for itself not for other packages, if I'm a `sdk` user I don't care about changes to `react`, etc